### PR TITLE
Solved Module undefined  error

### LINKED ISSUE #1094: Adding some modules from drop-down throws error
Adding certain modules(eg Paintbucket, overlay) from the drop-down throws error.
Screenshot from 2019-05-31 23-01-47

### PR DESCRIPTION
**Pull request by aashna27**
Fixes #1094 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask @publiclab/is-reviewers  for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
Hmm, shouldn't we make all module names `lower-case-hyphenated-words` without `Spaces Between`? That would flatten and simplify things... and it's all right if that's in the HTML select UI, too, i think... what do you think, @aashna27 ?

**Comment by jywarren** [comment]
Ack! Mistaken mouse button press! 😨 🐁

**Comment by aashna27** [comment]
Yeah I agree lower case would look more uniform in the drop-down, i ll make
the changes tomorrow!! Thanks.😊😊

**Comment by jywarren** [comment]
thank you!!!

File: src/Modules.js [hunk 1]
```diff
@@ -31,8 +31,8 @@ module.exports = {
   'invert': require('image-sequencer-invert'),
```
Comment (aashna27) [inline src/Modules.js]: The invert module comes from our own image-sequencer npm so the lower-case wont be reflected until new update, and I let it remain that way as this was different, using our repository through node_modules.

**Comment by aashna27** [comment]
🤣 Yeah we wish! And this one was caused due to no space between two words
ex gridoverlay, in the info.json

**Comment by jywarren** [comment]
Hi @aashna27 this should be good to merge if you can resolve the conflict! Thank you!!!!

**Comment by aashna27** [comment]
Yes i ll just resolve it.

**Comment by aashna27** [comment]
Yeah harsh i ll look into it in like 1 hour, i hope it will be okay!

**Comment by aashna27** [comment]
Resolved conflicts!

**Comment by jywarren** [comment]
Super!!!

**Comment by HarshKhandeparkar** [comment]
@jywarren @aashna27 check out the demo,  these changes are live

**Pull request merged**